Miscellaneous/scf-dictionary

```diff
@@ -0,0 +1,9 @@
+scf_guess = {
+    "EthaneA c-c": "8 molecule, 7 guess"
+    "PropaneA c-c-c": "8 molecule, 8 guess"
+    "PropaneA cc-c": "8 molecule, 8 guess"
+    "ButaneA c-c-c-c": "10 molecule, 10 guess"
+    "ButaneA cc-cc": "10 molecule, 10 guess"
+    "ButaneA ccc-c": "10 molecule, 10 guess"
+}
```

Comment:
Instead of manually entering  the data into the dictionary like this you should have your scripts do it. i'd do something like:
```.py
scf_guess = {}

# Read scf_guess from a file (look up pickling)

if "EthaneA c-c" not in scf_guess:
    funct_groups = # Set up 
    scf_guess["EthaneA c-c"] = guess_driver("EthaneA.sdf", funct_grps, [1,1])

if "PropaneA c-c-c" not in scf_guess:
    funct_group = # setup
    scf_guess["PropaneA c-c-c"] = guess_driver("PropaneA.sdf", funct_grps, [1,2,1])

# other data points follow

# Save scf guess to a file (look up pickling)
```